.github/workflows/ci.yml

```diff
@@ -35,9 +35,15 @@ jobs:
       - name: Tests
         run: bundle exec rake test
 
+  frontend:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 14
```

Comment:
Why the downgrade?

Reply:
The build isn't compatible with node 16 and later. I need to update half of the deps 😄 and then the other half than won't be compatible with the updated first half 🤣